src/link_rfu_3.c

```diff
@@ -879,7 +892,7 @@ void RecordMixTrainerNames(void)
         s32 connectedTrainerRecordIndices[5];
         struct TrainerNameRecord *newRecords = AllocZeroed(20 * sizeof(struct TrainerNameRecord));
 
-        // Check if we already have a record saved for connected trainers.
+        // Check if we alsendy have a record saved for connected trainers.
```

Comment:
Why is this changed (and the below one)?

Reply:
`s/read/send/g` 🤦‍♂